### PR TITLE
ReadOnlyMemoryContent class description

### PR DESCRIPTION
**Pull request by aik-jahoda**
## Summary

Collect all changes from #3916 related to ReadOnlyMemoryContent

### CODE REVIEW

File: xml/System.Net.Http/ReadOnlyMemoryContent.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Provides HTTP content based on a ReadOnlyMemory&lt;System.byte&gt;.</summary>
```
Comment (gewarren) [inline xml/System.Net.Http/ReadOnlyMemoryContent.xml:26]:
```suggestion
    <summary>Provides HTTP content based on a <see cref="T:System.ReadOnlyMemory&lt;System.byte&gt;" />.</summary>
```

Comment (karelz) [reply xml/System.Net.Http/ReadOnlyMemoryContent.xml:26]: `System.byte` should be `System.Byte` with capital 'B'.

File: xml/System.Net.Http/ReadOnlyMemoryContent.xml [hunk 2]
```diff
@@ -49,8 +49,8 @@
         <Parameter Name="content" Type="System.ReadOnlyMemory&lt;System.Byte&gt;" />
       </Parameters>
       <Docs>
-        <param name="content">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="content">The content used to initialize the <see cref="T:System.Net.Http.ReadOnlyMemoryContent" />.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Net.Http.ReadOnlyMemoryContent" /> class.</summary>
         <remarks>To be added.</remarks>
```
Comment (ManickaP) [inline xml/System.Net.Http/ReadOnlyMemoryContent.xml:54]: Do we leave `<remarks>`, which are not provide and should stay empty, as "To be added"?

Comment (aik-jahoda) [reply xml/System.Net.Http/ReadOnlyMemoryContent.xml:54]: It seems the "To be added." tags are not rendered in doc. @gewarren can you please confirm?

Comment (gewarren) [reply xml/System.Net.Http/ReadOnlyMemoryContent.xml:54]: That's correct. "To be added." is recognized by the build process and omitted.

**Pull request merged**